### DISCUSSION TITLE
Rule refactor

### DISCUSSION THREAD

**Discussion by chu121su12** [unocss/unocss#346] [Ideas]
@antfu Should the rules with simple single return be simplified from

```ts
  [/^ring-offset-(.+)$/, ([, d]) => {
    const value = h.px(d || '1')
    if (value)
      return { '--un-ring-offset-width': value }
  }],
```

to

```ts
  [/^ring-offset-(.+)$/, ([, d]) => ({ '--un-ring-offset-width': h.px(d || '1') })],
```

?

Removing the if-statement, the rule will be removed here instead
https://github.com/antfu/unocss/blob/ba752d334a352cb421c10006d713227c6139c296/packages/core/src/utils/object.ts#L34-L41

Or shoud the code be changed back to using if-statement?

**Comment by antfu** [comment]
yes we could remove the `if`
